acceptance/acceptance_test.go

```diff
@@ -28,6 +27,7 @@ import (
 	"github.com/sclevine/spec"
 	"github.com/sclevine/spec/report"
 
+	"github.com/buildpack/lifecycle/metadata"
```

Comment:
This should be in the second group of imports (technically it's third-party to pack)